tests/console/verify_efi_mok.pm

```diff
@@ -149,7 +149,7 @@ sub get_esp_info {
         if (!defined($drive) && $line =~ /gpt/ && $line =~ /$vbd/) {
             ($drive) = split(/:/, $line, 2);
         }
-        if (!defined($esp_part_no) && $line =~ /boot,\s?esp/) {
+        if (!defined($esp_part_no) && $line =~ /boot,\s?esp|boot/) {
```

Comment:
Could use a comment that this is due to ancient parted in 12 SP5 not detecting ESP types

Reply:
Comment added.